src/temperature_render.jl

```diff
@@ -0,0 +1,67 @@
+using GeodesicRendering
+using AccretionGeometry
+using CarterBoyerLindquist
+using .AccretionFormulae
```

Comment:
Shouldn't need a `.` here if the environment is correctly configured. For scripts, you can `Pkg> add ./path/to/AccretionFormulae.jl` to add it as a temporary dependency.

Reply:
I tried this, and have run into these errors, am I missing something obvious? I've left it in for the time being.

```julia

(DevEnv) pkg> add add .C:\Users\Dan\Documents\Uni\Year 4\Project\GitHub\Githubbackup\AccretionFormulae.jl\src\AccretionFormulae.jl
ERROR: Unable to parse `.C:\Users\Dan\Documents\Uni\Year` as a package.

(DevEnv) pkg> add .C:\Users\Dan\Documents\Uni\Year 4\Project\GitHub\AccretionFormulae.jl\src\AccretionFormulae.jl
ERROR: Unable to parse `.C:\Users\Dan\Documents\Uni\Year` as a package.

(DevEnv) pkg> add .AccretionFormulae.jl
ERROR: Unable to parse `.AccretionFormulae.jl` as a package.

(DevEnv) pkg> add AccretionFormulae.jl
    Updating registry at `C:\Users\Dan\.julia\registries\AstroRegistry`
    Updating git-repo `https://github.com/astro-group-bristol/AstroRegistry`
    Updating registry at `C:\Users\Dan\.julia\registries\General`
    Updating git-repo `https://github.com/JuliaRegistries/General.git`
   Resolving package versions...
ERROR: expected package `AccretionFormulae [31bec1c3]` to be registered
```